submit.py

```
@@ -0,0 +1,53 @@
+    print(result.stdout.decode())
+    pattern = r"TODO-(\d+)"
+    result = re.search(pattern, result.stdout.decode())
+    search = "TODO" + ":"
+
+    if result:
+        issue = result.group(0)
+        number = int(result.group(1))
+        print(f"Created issue {issue}")
+        command = f"sed -i '{data['line']}s/{search}/TODO-{number}:/' {data['filename']}"
+        print(f"Running command: {command}")
+        result = subprocess.run(command, shell=True, capture_output=True)
```

Ruff (0.8.2)

`subprocess` call with `shell=True` identified, security issue

(S602)